satellite/marketing/marketingweb/server.go

```diff
@@ -53,20 +71,22 @@ func (s *Server) commonPages() []string {
 }
 
 // NewServer creates new instance of offersweb server
-func NewServer(logger *zap.Logger, config Config, listener net.Listener) (*Server, error) {
+func NewServer(logger *zap.Logger, config Config, service *marketing.Service, listener net.Listener) (*Server, error) {
 	s := &Server{
 		log:      logger,
 		config:   config,
 		listener: listener,
+		service:  service,
 	}
 
 	logger.Sugar().Debugf("Starting Marketing Admin UI on %s...", s.listener.Addr().String())
-
 	fs := http.StripPrefix("/static/", http.FileServer(http.Dir(s.config.StaticDir)))
 	mux := mux.NewRouter()
 	if s.config.StaticDir != "" {
+		mux.HandleFunc("/", s.getOffers)
 		mux.PathPrefix("/static/").Handler(fs)
-		mux.Handle("/", s)
+		mux.HandleFunc("/create-free-credit", s.createOffer)
```

Comment:
Are `/create-free-credit`  and `/create-ref-offer` supposed to link to the same handler?

Reply:
Yes, there is logic in the handler that assigns the offer type for newly created offer based on the endpoint the request came from.

Reply:
I pushed an update so only one endpoint is used for create which takes an `{offer_type}` param which is then used in a conditional in the handler to set the offer type https://github.com/storj/storj/pull/2186#discussion_r293925878.